x-pack/test/rule_registry/common/lib/authentication/roles.ts

```diff
@@ -0,0 +1,598 @@
+ * will exist when the spaces plugin is disabled.
+ */
+export const rolesDefaultSpace = [
+  noKibanaPrivileges,
+  globalRead,
+  securitySolutionOnlyAllSpacesAll,
+  securitySolutionOnlyReadSpacesAll,
+  observabilityOnlyAllSpacesAll,
+  observabilityOnlyReadSpacesAll,
+];
+
+/**
+ * These roles are only to be used in the 'trial' tests
+ * since they rely on subfeature privileges which are a gold licencse feature
+ * maybe put these roles into a separate roles file like "trial_roles"?
+ */
+export const observabilityMinReadAlertsRead: Role = {
+  name: 'obs_only_alerts_read',
+  privileges: {
+    elasticsearch: {
+      indices: [
+        {
+          names: ['*'],
+          privileges: ['all'],
+        },
+      ],
+    },
+    kibana: [
+      {
+        feature: {
+          apm: ['minimal_read', 'alerts_read'],
+          ruleRegistry: ['all'],
+          actions: ['read'],
+          builtInAlerts: ['all'],
+          alerting: ['all'],
+        },
+        spaces: ['space1'],
+      },
+    ],
+  },
+};
+
+export const observabilityMinReadAlertsReadSpacesAll: Role = {
+  name: 'obs_minimal_read_alerts_read_spaces_all',
+  privileges: {
+    elasticsearch: {
+      indices: [
+        {
+          names: ['*'],
+          privileges: ['all'],
+        },
+      ],
+    },
+    kibana: [
+      {
+        feature: {
+          apm: ['minimal_read', 'alerts_read'],
+          ruleRegistry: ['all'],
+          actions: ['read'],
+          builtInAlerts: ['all'],
+          alerting: ['all'],
+        },
+        spaces: ['*'],
+      },
+    ],
+  },
+};
+
+export const observabilityMinimalRead: Role = {
+  name: 'obs_minimal_read_spaces_all',
+  privileges: {
+    elasticsearch: {
+      indices: [
+        {
+          names: ['*'],
+          privileges: ['all'],
+        },
+      ],
+    },
+    kibana: [
+      {
+        feature: {
+          apm: ['minimal_read'],
+          ruleRegistry: ['all'],
+          actions: ['read'],
+          builtInAlerts: ['all'],
+          alerting: ['all'],
+        },
+        spaces: ['space1'],
+      },
+    ],
+  },
+};
+
+export const observabilityMinimalReadSpacesAll: Role = {
+  name: 'obs_minimal_read_spaces_all',
+  privileges: {
+    elasticsearch: {
+      indices: [
+        {
+          names: ['*'],
+          privileges: ['all'],
+        },
+      ],
+    },
+    kibana: [
+      {
+        feature: {
+          apm: ['minimal_read'],
+          ruleRegistry: ['all'],
+          actions: ['read'],
+          builtInAlerts: ['all'],
+          alerting: ['all'],
+        },
+        spaces: ['*'],
+      },
+    ],
+  },
+};
+
+export const observabilityOnlyAlertsRead: Role = {
+  name: 'obs_alerts_read_spaces_all',
+  privileges: {
+    elasticsearch: {
+      indices: [
+        {
+          names: ['*'],
+          privileges: ['all'],
+        },
+      ],
+    },
+    kibana: [
+      {
+        feature: {
+          apm: ['alerts_read'],
```

Comment:
This isn't a realistic test scenario. A role can't grant the `alerts_read` sub-feature privilege without `minimal_read` or `minimal_all`.

Since you already have the `observabilityMinReadAlertsRead` role defined above, I think this role and its associated test cases can be removed.